web/src/main/java/org/wso2/testgrid/web/api/TestPlanService.java

```diff
@@ -176,17 +176,17 @@ public Response getLogContent(@PathParam("id") String id, @QueryParam("truncate"
         } catch (TestGridDAOException e) {
             String msg = "Error occurred while fetching the TestPlan by id : '" + id + "'";
             logger.error(msg, e);
-            return Response.serverError()
+            return Response.serverError().status(Response.Status.INTERNAL_SERVER_ERROR)
```

Comment:
Both #serverError, or set the status code to 500 does the same thing. [1] I'm not sure why we need to do the same thing twice!

http://grepcode.com/file/repo1.maven.org/maven2/javax.ws.rs/jsr311-api/1.0/javax/ws/rs/core/Response.java#199